### PR TITLE
Disable merging new testcases, dictionary analysis for libClusterFuzz

### PR DESCRIPTION
**Pull request by oliverchang**
For #2400.

### CODE REVIEW

File: src/clusterfuzz/_internal/bot/fuzzers/libFuzzer/engine.py [hunk 1]
```diff
@@ -322,11 +327,13 @@ def fuzz(self, target_path, options, reproducers_dir, max_time):
     })
 
     # Remove fuzzing arguments before merge and dictionary analysis step.
-    merge_arguments = options.arguments[:]
-    libfuzzer.remove_fuzzing_arguments(merge_arguments, is_merge=True)
-    self._merge_new_units(target_path, options.corpus_dir, new_corpus_dir,
-                          options.fuzz_corpus_dirs, merge_arguments,
-                          parsed_stats)
+    non_fuzz_arguments = options.arguments[:]
```
Comment (jonathanmetzman) [inline src/clusterfuzz/_internal/bot/fuzzers/libFuzzer/engine.py:330]: Maybe change to `.copy()`?

**Review by jonathanmetzman — COMMENTED** [review]
Thanks!
LGTM

File: src/clusterfuzz/_internal/bot/fuzzers/libFuzzer/engine.py (hunk 1 above)
Comment (oliverchang) [reply src/clusterfuzz/_internal/bot/fuzzers/libFuzzer/engine.py]: done!

File: src/clusterfuzz/_internal/bot/fuzzers/libFuzzer/engine.py [hunk 2]
```diff
@@ -322,11 +327,13 @@ def fuzz(self, target_path, options, reproducers_dir, max_time):
...
+    libfuzzer.remove_fuzzing_arguments(non_fuzz_arguments, is_merge=True)
```
[the remark this answers is not in the document]
Comment (oliverchang) [reply src/clusterfuzz/_internal/bot/fuzzers/libFuzzer/engine.py:331]: yeah, but that would be a bigger change beyond the scope of this PR. need to clean this up later.

**Pull request merged**